docs/getting-started/server/admin.md

```diff
@@ -1,19 +1,51 @@
-# Admin Portal
+# Bitwarden Portal
 
-## Configuring users
+## Configuring access
 
-Admin Portal authentication is done entirely through a passwordless flow, using a link sent through
-email. The email address must be listed in the `adminSettings:admins` user secret to be authorized.
+### Authentication
+
+Bitwarden Portal authentication is done entirely through a passwordless flow, using a link sent
+through email. The email address must be listed in the `adminSettings:admins` user secret to be
```

Comment:
This needs to be updated.

Reply:
Got it.  I've updated the `adminSettings:admins` as well to include all the emails.